hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/FileIOEngine.java

```diff
@@ -267,6 +310,76 @@ void refreshFileConnection(int accessFileNum, IOException ioe) throws IOExceptio
     }
   }
 
+  /**
+   * Delete bucketcache files
+   */
+  @Override
+  public void deleteCacheDataFile() {
+    if (filePaths == null) {
+      return;
+    }
+    for (String file : filePaths) {
+      new File(file).delete();
+    }
+  }
+
+  @Override
+  public byte[] calculateChecksum(String algorithm) throws IOException{
+    if (filePaths == null) {
+      return null;
+    }
+    StringBuilder sb = new StringBuilder();
+    for (String filePath : filePaths){
+      File file = new File(filePath);
+      if (file.exists()){
+        sb.append(filePath);
+        sb.append(getFileSize(filePath));
+        sb.append(file.lastModified());
+      } else {
+        throw new IOException("Cache file: " + filePath + " is not exists.");
```

Comment:
In fact, after FileIOE inited, the file path must be exist anyway. I know's it's the complaining in code check, so can we just swallow this IOE in this method, instead of throw to upper caller?